pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1246,20 +1246,19 @@ func (b *MasqueradeBindMechanism) createNatRulesUsingNftables(proto iptables.Pro
 			if err != nil {
 				return err
 			}
-			err = b.handler.NftablesAppendRule(proto, "nat", "KUBEVIRT_POSTINBOUND",
-				b.handler.GetNFTIPString(proto), "saddr", getEnvoyLoopbackAddress(),
-				strings.ToLower(port.Protocol),
-				"dport",
-				strconv.Itoa(int(port.Port)),
-				"counter", "snat", "to", b.getGatewayByProtocol(proto))
-			if err != nil {
-				return err
-			}
 		}
 	}
 	return nil
 }
 
+func getSrcAddressesToSnat(vmi *v1.VirtualMachineInstance, proto iptables.Protocol) string {
```

Comment:
No reason in particular, I didn't add the masqueradeBind receiver because it's not used.
Similarly, `getLoopbackAddress` doesn't have it, although I see that this method is more specific for MasqueradeBind.
Do you think it should be added to express the relation?

Reply:
Well, you don't use the receiver, but you could - instead of using the vmi input parameter, you could use the member of the receiver struct.

Anyway. These are peanuts.

Reply:
That's a good point actually, I missed that. I like the idea of these method having the same parameters

Reply:
[done](https://github.com/kubevirt/kubevirt/compare/af0422b4fea220c2fe7c0a639d4b60844018199d..beac357b4698e4f150480521e6303d22bd2e30ae)